pkg/cmd/roachtest/test_runner.go

```diff
@@ -580,6 +580,10 @@ func (r *testRunner) runWorker(
 		}
 	}()
 
+	work.mu.Lock()
+	totalTestCount := work.count * len(work.mu.tests)
```

Comment:
This seems like it's only counting the number of tests given to a single worker, but we want to do this over the entire nightly run. 

https://github.com/cockroachdb/cockroach/blob/244b1a1b60c399b035377644cf09a0df53d2e066/pkg/cmd/roachtest/run.go#L114-L119

We already figure out the number of tests here, so i don't think we need to do anything special, just use that.